bin/demo.py

```diff
@@ -31,97 +33,75 @@
 pass or fail. The duration of each job will be between 120 and 240 seconds.
 """
 
-import json
+import dcos
 import os
 import random
-import requests
-import shutil
 
 from docopt import docopt
-from subprocess import call, CalledProcessError, check_output
-from urlparse import urlparse
+from subprocess import call
+from urllib.parse import urlparse
+
+from shakedown import *
 
 def log(message):
-    print "[demo] {}".format(message)
+    print("[demo]: {}".format(message))
```

Comment:
We could, but I don't think it offers any advantages over `print` in this use case (since no special [colorized, extended charset] characters are being printed).